### PR TITLE
Advanced search hello world

### PR DESCRIPTION
**Pull request by alex-hornet**

### CODE REVIEW

File: angular-spa-ui/webpack.config.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ module.exports = {
         vendor: [
             'underscore',
             'angular',
-            'angular-ui-router',
-            'bootstrap-sass!./bootstrap-sass.config.js'
+            'angular-ui-router'
+            //'bootstrap-sass!./bootstrap-sass.config.js'
```
Comment (tarhunakau) [inline angular-spa-ui/webpack.config.js]: 1 Check if PR #55 fixes it

File: angular-spa-ui/js/search-app/search-module/index.js [hunk 2]
```diff
@@ -1,15 +1,22 @@
 "use strict";
 
-require('angular-ui-router');
+module.exports = function(ngModule) {
+    
+    var advancedSearchModule = require('./advanced-search')(ngModule);
+    var searchModule = ngModule.module('searchApp.searchModule', ['ui.router', 'searchApp.searchModule.advancedSearch']);
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/index.js:6]: 2 Check do you really need "searchApp.searchModule.advancedSearch" for normal running the aplpication

**Comment by tarhunakau** [comment]
After fixes 1 and 2 could be merged

File: angular-spa-ui/js/search-app/search-module/index.js (hunk 2 above)
Comment (alex-hornet) [inline angular-spa-ui/js/search-app/search-module/index.js:6]: Yes, we need it.

**Comment by tarhunakau** [comment]
Could be merged

**Pull request merged**